code/workspaces/web-app/src/components/common/form/UserMultiSelect.js

```diff
@@ -17,7 +18,7 @@ function UserMultiSelect({ input, meta = null, fixedOptions = [], ...custom }) {
           meta,
           currentValue,
           setCurrentValue,
-          options: sortedUsers,
+          options: sortedUsers.filter(user => user.verified),
```

Comment:
This is good for filtering verified users.

I wonder though if we could add the project filtering now too? Elsewhere we use the projectUsersHook which could be used here to pull in only users on the project. The would be that this component is also used outside of the project context (asset repo). Not a serious issue though as this is how the system currently works.

Reply:
I've added ED-95 to the backlog to capture this and the investigation we did.